src/ostorlab/utils/system_info_helper.py

```diff
@@ -0,0 +1,96 @@
+"""Helper functions to get system information."""
+from dataclasses import dataclass
+from typing import Optional
+
+
+@dataclass
+class CpuInfo:
+    """CPU information."""
+
+    load: list[float]
+
+
+@dataclass
+class MemoryInfo:
+    """Memory information."""
+
+    total: int
+    available: int
+    used: int
+    percent: float
+
+
+@dataclass
+class StorageInfo:
+    """Storage information."""
+
+    total: int
+    used: int
+    free: int
+
+
+@dataclass
+class SystemLoadInfo:
+    """System load information."""
+
+    cpu_load: CpuInfo
+    memory_usage: MemoryInfo
+    storage: StorageInfo
+
+    def __str__(self) -> str:
+        """String representation of the system load information."""
+        return (
+            f"CPU Load: {self.cpu_load.load}\n"
+            f"Memory Usage: Total={self.memory_usage.total}, "
+            f"Available={self.memory_usage.available}, "
+            f"Used={self.memory_usage.used}, "
+            f"Percent={self.memory_usage.percent}\n"
+            f"Storage: Total={self.storage.total}, "
+            f"Used={self.storage.used}, "
+            f"Free={self.storage.free}"
+        )
+
+
+def get_system_load() -> Optional[SystemLoadInfo]:
+    """Get system load information.
+    We use psutil to get the system load information. in case `psutil` is not available, we return None.
+    Returns:
+        System load information or None in case `psutil` is not available.
+    """
```

Comment:
```suggestion
    """Get system load information.
    
    We use psutil to get the system load information. in case `psutil` is not available, we return None.
    
    Returns:
        System load information or None in case `psutil` is not available.
    """
```